src/test/unit/requests/dailyCauseListRequests.test.ts

```diff
@@ -0,0 +1,105 @@
+import sinon from 'sinon';
+import {dataManagementApi} from '../../../main/resources/requests/utils/axiosConfig';
+import {DailyCauseListRequests} from '../../../main/resources/requests/dailyCauseListRequests';
+import fs from 'fs';
+import path from 'path';
+
+const dailyCauseListRequests = new DailyCauseListRequests();
+
+const rawData = fs.readFileSync(path.resolve(__dirname, '../mocks/dailyCauseList.json'), 'utf-8');
+const dailyCauseListData = JSON.parse(rawData);
+
+const rawMetaData = fs.readFileSync(path.resolve(__dirname, '../mocks/dailyCauseListMetaData.json'), 'utf-8');
+const metaData = JSON.parse(rawMetaData);
+
+const stub = sinon.stub(dataManagementApi, 'get');
+
+const artefactId = 'abc';
+
+const errorResponse = {
+  response: {
+    data: 'test error',
+  },
+};
+const errorRequest = {
+  request: 'test error',
+};
+
+const errorMessage = {
+  message: 'test',
+};
+
+describe('getDailyCauseList()', () => {
+  beforeEach(() => {
+    stub.withArgs('/publication/abc1/payload').resolves(Promise.reject(errorResponse));
+    stub.withArgs('/publication/abc2/payload').resolves(Promise.reject(errorRequest));
+    stub.withArgs('/publication/abc3/payload').resolves(Promise.reject(errorMessage));
+    stub.withArgs('/publication/'+ artefactId + '/payload').resolves({data: dailyCauseListData});
```

Comment:
none of these need to live in the before each, you dont need a before each if youre only setting stubs, youre resetting the same thing each test, same with all the before each's youve done this in